tikv-client/src/main/java/com/pingcap/tikv/meta/TiTableInfo.java

```diff
@@ -255,6 +262,15 @@ public List<TiIndexInfo> getIndices(boolean includingAll) {
     }
   }
 
+  public TiIndexInfo getPrimaryKey() {
```

Comment:
Can you test if we can getPrimaryKey in pkIsHandle.
pkIsHandle's PrimaryKey is not in the indices in my memory.